wbooks/src/app/screens/dashboard/screens/Home/components/SearchBar/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './styles.css';
+import searchSvg from './assets/search.svg'
+
+class SearchBar extends React.Component {
+
+  handleSubmit = (event) => {
```

Comment:
When there is a single argument, we prefer to write that arg without parenthesis:
`handleSubmit = event => {`